packages/roosterjs-editor-core/lib/utils/getCursorRect.ts

```diff
@@ -19,11 +19,14 @@ function getRectFromClientRect(clientRect: ClientRect): Rect {
         : null;
 }
 
-// Returns a rect representing the location of the cursor.
-// In case there is a uncollapsed selection witin editor, this returns
-// the position for focus node.
-// The returned rect structure has a left and right and they should be same
-// here since it is for cursor, not for a range.
+/**
+ * Returns a rect representing the location of the cursor.
```

Comment:
Some places are using "Return", some are using "Returns". Please use the same one.

Reply:
Use 'Returns' for all.